integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/sort/FieldSearchSortIT.java

```diff
@@ -171,6 +171,42 @@ public void byField() {
 		}
 	}
 
+	@Test
+	public void byField_use_alreadyExists_missingValue() {
```

Comment:
Please add `@TestForIssue`: this test is very specific and a bit obscure, we may need to come back to the ticket and PR one day.